checklist.md

```diff
@@ -31,17 +31,17 @@ Same goes for files that you use in tests, **let's put them into this folder:**
 #### Try to cover different scenarios in tests
 Your task is to include edge cases apart from the regular method use case.
 
-#### Don't test the Main class
-We want to test only business logic, so there is no need to cover the `Main` class with tests. 
-You can exclude the Main class with the `main()` method from being checked for code coverage in `pom.xml`.   
+#### Don't test the main class
```

Comment:
always review what you're committing, to avoid unintended changes, be careful with project-wide replacements